README.md

```diff
@@ -49,10 +47,10 @@ See our [example](./examples) notebooks for how to develop with `gordo` locally.
 ---
 
 ## Install: 
-`pip install --upgrade gordo-components`  
+`pip install --upgrade gordo`  
 
 Bleeding edge:  
-`pip install git+https://github.com/equinor/gordo-components.git`
+`pip install git+https://github.com/equinor/gordo.git`
```

Comment:
You want to have this in now, or in a separate one, after we move it?

Reply:
:man_shrugging: 
I guess my mind's eye was thinking we'd make a release, merge this, and then immediately update the url and others like readthedocs/codecov ect?